src/celeritas/em/xs/WentzelXsCalculator.hh

```diff
@@ -0,0 +1,109 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/xs/WentzelXsCalculator.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+#include "corecel/math/Algorithms.hh"
+#include "celeritas/phys/AtomicNumber.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Calculate the ratio of the electron to total Wentzel cross sections for
+ * elastic Coulomb scattering.
+ */
+class WentzelXsCalculator
+{
+  public:
+    // Construct the calculator from the given values
+    inline CELER_FUNCTION WentzelXsCalculator(AtomicNumber target_z,
+                                              real_type screening_coefficient,
+                                              real_type cos_t_max_elec);
+
+    // The ratio of electron to total cross section for Coulomb scattering
+    inline CELER_FUNCTION real_type operator()() const;
```

Comment:
If the call operator takes no arguments and is "const" (doesn't alter anything's state) then there's not much advantage to it being a class. This whole class might be better as a helper function, unless you want to be able to reuse the same screening/cosine factors and then calculate the ratio for a particular target z: in that case, `target_z` would be the argument to `operator()`.

Refactoring as a function shouldn't alter the performance, it just might make the code clearer/shorter, since each of the helper functions are just a couple of lines.

Reply:
For the WentzelVI msc model it would be useful to be able to reuse this for different Z (though the screening coefficient also depends on Z). We'll also need the total xs instead of a ratio for WentzelVI, so this may need to be refactored a bit eventually.

Reply:
Does the MSC model use the same Moliere screening coefficient? If so, I can move that calculation to the `WentzelXsCalculator` so that it'd be readily available.

Reply:
@hhollenb The screen factor should be common for both MSC and SingleScattering (SS), but this cross section ratio of SS may not be used for MSC as you already refactored (simplified ) the ratio for SS.